spec/lib/services/handle_submission_for_store_spec.rb

```diff
@@ -44,7 +44,7 @@
 
     describe "#call" do
       def stable_as_json(record)
-        record.as_json(except: %i[id created_at updated_at])
+        record.as_json(except: %i[id created_at updated_at flipper_admin_access])
```

Comment:
If we delete this field then wouldnt have to do this aswell

Reply:
responded above